frontend/src/Containers/Home/index.js

```diff
@@ -43,9 +47,41 @@ class Home extends Component {
     }
     this.setState({ ingredients })
   }
+  handleRecipeClick(recipeId) {
+    this.props.selectRecipe(recipeId)
+    this.setState({ selectedRecipeId: recipeId })
+  }
   render() {
     const { term, ingredients } = this.state
-    const { recipes, isLoading } = this.props
+    const { recipes, isLoading, selectedRecipeId } = this.props
+    const ListItem = withStyles({
+      root: {
+        "&$selected": {
+          backgroundColor: "steelblue",
+          color: "white",
+          "& .MuiListItemIcon-root": {
+            color: "white",
+          },
+        },
+        "&$selected:hover": {
+          backgroundColor: "steelblue",
+          color: "white",
+          "& .MuiListItemIcon-root": {
+            color: "white",
+          },
+          cursor: "pointer",
+        },
+        "&:hover": {
+          backgroundColor: "lightgray",
+          color: "white",
+          "& .MuiListItemIcon-root": {
+            color: "white",
+          },
+          cursor: "pointer",
+        },
+      },
+      selected: {},
+    })(MuiListItem)
```

Comment:
I wanted to use my own styled ListItem to have somewhat better UI interactions by making each recipe appear clickable.